src/lib/dfp/on-slot-viewable.ts

```diff
@@ -38,10 +38,17 @@ const setAdSlotMinHeight = (advert: Advert): void => {
 
 	const isStandardAdSize = !size.isProxy();
 	if (isStandardAdSize) {
-		const adSlotHeight = size.height + AD_LABEL_HEIGHT;
-		void fastdom.mutate(() => {
-			node.style.minHeight = `${adSlotHeight}px`;
-		});
+		void fastdom
+			.measure(() => node.getAttribute('data-label') === 'true')
+			.then((hasLabel) => {
+				const labelHeight = hasLabel ? AD_LABEL_HEIGHT : 0;
+				if (hasLabel) {
```

Comment:
Can this `if` condition be removed now?

Reply:
Just missed this - addressed here:
https://github.com/guardian/commercial/pull/989